content/documentation/pages/6-recipes/8-batch/0-index.md

```diff
@@ -0,0 +1,10 @@
+---
+path: 'recipes/file-ingest/'
+title: 'Batch'
+description: 'Using Spring Cloud Data Flow with Spring Batch'
+summary: true
+---
+
+This section provides recipes specific to using Spring Batch with Spring Cloud Data Flow:
```

Comment:
This doesn't render the same way the other recipes on this page does and the path doesn't make sense (`recipes/file-ingest/`).